specs/README.md

```diff
@@ -5,19 +5,20 @@ Here you'll find specifications for custom elements and other library features.
 > **Note:** When authoring a new spec, please use [our standard spec template](./template.md). Specs can be submitted in the form of a PR. Prior to creating a PR, be sure to remove any boilerplate from the spec template which was not used. The PR that introduces your spec should also add a link to this readme in the specs section below.
 
 ## Specs
-- [Badge](./badge.md)
-- [Card](./card/card.md)
-- [Checkbox](./checkbox.md)
-- [Dialog](./dialog/dialog.md)
-- [Flipper](./flipper/flipper.md)
-- [Form associated custom elements]("./form-associated-custom-element.md")
+- [Anchored Region](../packages/web-components/fast-components/src/anchored-region/anchored-region.spec.md)
```

Comment:
Historically we haven't capitalized the second word of a component name.  "Anchored region", although, I much prefer what you have here, it hasn't been our standard.